osquery/remote/utility.h

```diff
@@ -225,15 +244,18 @@ class TLSRequestHelper : private boost::noncopyable {
    */
   template <class TSerializer>
   static Status go(const std::string& uri,
-                   boost::property_tree::ptree& params,
+                   JSON& params,
                    std::string& output,
                    const size_t attempts) {
     Status s;
+    JSON override_params;
+    const auto& params_doc = params.doc();
+    const auto& override_params_doc = override_params.doc();
 
-    boost::property_tree::ptree override_params;
-    for (const auto& param : params) {
-      if (param.first.find('_') == 0) {
-        override_params.put(param.first, param.second.data());
+    for (auto& m : params_doc.GetObject()) {
+      std::string name = m.name.GetString();
+      if (name.find('_') == 0) {
```

Comment:
std::string::npos ?

Reply:
This is a `starts_with` so I think it should be fine like this no?

Reply:
My bad